src/components/form-elements/pages/select-interactive.jsx

```diff
@@ -38,6 +45,10 @@ const Selects = () => {
       <DocsActiveBlock backgroundColor="dark" settings={settings}>
         <Select options={exampleOptions} size="m" color="white" />
       </DocsActiveBlock>
+
+      <DocsActiveBlock backgroundColor="dark" settings={settings}>
+        <Select groupedOptions={exampleGroupedOptions} size="m" color="white" />
```

Comment:
@ensisf is this example still valid (`groupedOptions`)?

Reply:
Oh, sorry, I've missed here. I will in a moment

Reply:
Done